content/log/explore.md

```diff
@@ -0,0 +1,146 @@
+
+{{< img src="log/explore/timerange.png" alt="Timerange" style="width:75%;">}}
+
+## Log list
+The log list is the list of events that match the selected context, a context is defined by a [search bar](#search-bar) filter and a [time range](#time-range).
+{{< img src="log/explore/log_list.png" alt="Log List" >}}
+
+### Sorting the log list
+
+Click on the **date** columns to get your logs in an ascending or descending order in the Log list.
+
+### Filtering the log list
+If you enter a valid query into the [search bar](#search-bar),  words that match your query are highlighted and log displayed match your facet criterias:
+
+Searching for all the logs containing the word "deleted" from host:i-0223afcd9cb2fa6a2 : 
+{{< img src="log/explore/log_list_highlighted.png" alt="Log List highlighted" >}}
+
+### Displaying a full log
+To display detail of a log line, just click on it:
+
+{{< img src="log/explore/log_in_log_list.png" alt="Log in log list" >}}
+
+### View a log line in context
+When displaying the detail of a log line, ask to « view in context ». 
+It removes any filters from your search bar and display log lines preceding this line and the few log lines following it.
+
+If you send the log lines in syslog format, the hostname and app name corresponding to the log line are automatically be selected.
+
+## Columns
+In order to get more details about the objects you are looking at, select a few facets to display with the **Columns** button:
+{{< img src="log/explore/log_list_with_columns.png" alt="Log List with columns" >}}
+
+## Facets 
+
+A facet displays all the values expressed by an attribute or a tag. This is also a handle to easily filter over your data.
+
+A Facet helps you to break down your datasets over user(s), service(s), etc ...
+
+### Create a Facet
+
+To start using an attribute or a tag in a Facet or in the search, you simply need to click on it and add it as a Facet:
+{{< img src="log/explore/create_facet.png" alt="Create Facet" style="width:75%;">}}
+
+Once this is done, the value of this attribute for all new logs is  stored and can be used for search either in the search bar or in the Facet Panel
+
+### Facet Panel
+
+Use facets to easily filters on your logs. The search bar and url automatically reflect your selections.
+{{< img src="log/explore/facet_panel.png" alt="Facet panel" >}}
+
+### Expanded view
+
+Expand a facet to access advanced search options like 
+Including or excluding a pattern within a facet.
+
+For example: You want to have all host starting by “i-06”: 
+{{< img src="log/explore/expanded_view.png" alt="Expanded view" style="width:75%;" >}}
+
+## Search bar
+
+The search query language is based on the Lucene query string:
+
+[Apache Lucene - Query Parser Syntax](http://lucene.apache.org/core/2_9_4/queryparsersyntax.html)
+
+All search parameters are contained in the url, so it is very simple to share your view.
+
+### Search syntax
+A query is broken up into terms and operators.
```

Comment:
Some is missing here:

- Sequence " ... "
- Attributes starts with @...

Reply:
what's sequence?

Reply:
How about: "A query is composed of terms and operators."

Reply:
👍